docs/concepts/registries.md

```diff
@@ -23,6 +23,8 @@ public void registerBlocks(RegistryEvent.Register<Block> event) {
 
 The order in which `RegistryEvent.Register` events fire is arbitrary, with the exception that `Block` will *always* fire first, and `Item` will *always* fire second, right after `Block`. After the `Register<Block>` event has fired, all [`ObjectHolder`][ObjectHolder] annotations are refreshed, and after `Register<Item>` has fired they are refreshed again. They are refreshed for a third time after *all* of the other `Register` events have fired.
 
+`RegistryEvent`s are currently supported for the following types: `Block`, `Item`, `Potion`, `Biome`, `SoundEvent`, `PotionType`, `Enchantment`, `IRecipe`, `VillagerProfession`, `Entity`
```

Comment:
I got this list by looking at `net.minecraftforge.registries.GameData`. I'm not 100% certain the `Entity` is supported, but it seemed like that was the intent so I included it here. I don't have time to test it right now, but I intend to at some point over the next few days. If that is not accurate, it can certainly be removed.